utils/python/update_acme_tests.py

```diff
@@ -64,8 +64,8 @@
 
     "cime_developer" : (None,
                             ("NCK_Ld3.f45_g37_rx1.A",
-                             "ERI_D.f45_g37_rx1.A",
-                             "SEQ_Ld3.f09_g16.X",
+                             "ERI_D.f45_g37.X",
```

Comment:
The SEQ test makes more sense as an A case, but I wanted to make sure we were still testing an X case.  Also no need to run this test for 3 days, 9 steps works fine.